js/client/modules/@arangodb/testutils/testing.js

```diff
@@ -122,6 +122,7 @@ let optionsDocumentation = [
   '   - `disableClusterMonitor`: if set to false, an arangosh is started that will send',
   '                              keepalive requests to all cluster instances, and report on error',
   '   - `disableMonitor`: if set to true on windows, procdump will not be attached.',
+  '   - `enableAliveMonitor`: checks whether spawned processe disappear in midnight and aborts.',
```

Comment:
Why "in midnight"?

Reply:
`in the middle of executed tests` better?